htdocs/comm/propal/list.php

```diff
@@ -1911,7 +1915,11 @@
 			if (!$i) {
 				$totalarray['pos'][$totalarray['nbfield']] = 'p.total_tva';
 			}
-			$totalarray['val']['p.total_tva'] += $obj->total_tva;
+			if (empty($totalarray['val']['p.total_tva'])) {
```

Comment:
you can define `$totalarray['val']['p.total_tva'] = 0` at line 1589 and also the other